src/App.js

```diff
@@ -9,7 +9,6 @@ import useLocalStorageState from './hooks/useLocalStorageState';
 import Home from './components/Home';
```

Comment:
What did you change in the App.js?

Reply:
Hey @ciradu2204, I had to move Nav somewhere else because of the css issues

Reply:
okay, thanks